osgp/platform/osgp-domain-core/src/main/java/org/opensmartgridplatform/domain/core/entities/RtuDevice.java

```diff
@@ -21,6 +23,10 @@
     @Column(insertable = false)
     private Date lastCommunicationTime;
 
+    @ManyToOne()
+    @JoinColumn(name = "domain_info_id")
+    protected DomainInfo domainInfo;
```

Comment:
I don't think there is a need for this field to be `protected`. Can be changed to `private`.